tests/integration/test_cmorize_obs.py

```diff
@@ -0,0 +1,178 @@
+    coords_spec = [(time, 0), (zcoord, 1), (lats, 2), (lons, 3)]
+    cube = iris.cube.Cube(cube_data, dim_coords_and_dims=coords_spec)
+    return cube
+
+
+def put_dummy_data(data_path):
+    """Create a small dummy netCDF file to be cmorized."""
+    gen_cube = _create_sample_cube()
+    t_path = os.path.join(data_path, "woa13_decav81B0_t00_01.nc")
+    # correct var names
+    gen_cube.var_name = "t_an"
+    iris.save(gen_cube, t_path)
+    s_path = os.path.join(data_path, "woa13_decav81B0_s00_01.nc")
+    gen_cube.var_name = "s_an"
+    iris.save(gen_cube, s_path)
+    # incorrect var names
+    o2_path = os.path.join(data_path, "woa13_all_o00_01.nc")
+    gen_cube.var_name = "o2"
+    iris.save(gen_cube, o2_path)
+    no3_path = os.path.join(data_path, "woa13_all_n00_01.nc")
+    gen_cube.var_name = "no3"
+    iris.save(gen_cube, no3_path)
+    po4_path = os.path.join(data_path, "woa13_all_p00_01.nc")
+    gen_cube.var_name = "po4"
+    iris.save(gen_cube, po4_path)
+    si_path = os.path.join(data_path, "woa13_all_i00_01.nc")
+    gen_cube.var_name = "si"
+    iris.save(gen_cube, si_path)
+
+
+def check_log_file(log_file, no_data=False):
+    """Check the cmorization log file."""
+    with open(log_file, 'r') as log:
+        if no_data:
+            cases = []
+            for line in log.readlines():
+                case = "Could not find raw data WOA" in line
+                cases.append(case)
+            assert True in cases
+        else:
+            cases = []
+            for line in log.readlines():
+                case = "Fixing data" in line
+                cases.append(case)
+            assert True in cases
+
+
+def check_output_exists(output_path):
+    """Check if cmorizer outputted."""
+    # eg Tier2/WOA/OBS_WOA_clim_2013v2_Omon_thetao_200001-200002.nc
+    output_files = os.listdir(output_path)
+    # ['OBS_WOA_clim_2013v2_Omon_thetao_200001-200002.nc',
+    # 'OBS_WOA_clim_2013v2_Omon_so_200001-200002.nc']
+    assert len(output_files) == 2
+    assert 'OBS_WOA_clim' in output_files[0]
+    assert '_thetao_' in output_files[0]
+    assert '_so_' in output_files[1]
+
+
+def check_conversion(output_path):
+    """Check basic cmorization."""
+    cube = iris.load_cube(os.path.join(output_path,
+                                       os.listdir(output_path)[0]))
+    assert cube.coord("time").units == Unit('days since 1950-1-1 00:00:00',
+                                            calendar='gregorian')
+    assert cube.coord("latitude").units == 'degrees'
+
+
+@contextlib.contextmanager
+def arguments(*args):
+    backup = sys.argv
+    sys.argv = list(args)
+    yield
+    sys.argv = backup
+
+
+def test_cmorize_obs_woa_no_data(tmp_path):
+    """Test for example run of cmorize_obs command."""
+
+    config_user_file = write_config_user_file(tmp_path)
+    os.makedirs(os.path.join(tmp_path, 'raw_stuff'))
+    os.makedirs(os.path.join(tmp_path, 'raw_stuff', 'Tier2'))
```

Comment:
```suggestion
    os.makedirs(os.path.join(tmp_path, 'raw_stuff', 'Tier2'))
```
will automatically make subdirectories for you